packages/components/datetime/stories/DateTime.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import type { Meta, Story } from '@storybook/react/types-6-0';
+
+import { DateTime } from '../src/DateTime';
+import type { DateTimeProps } from '../src/DateTime';
+
+export default {
+  title: 'Components/DateTime',
+  component: DateTime,
+  parameters: {
+    controls: {
+      matchers: {
+        date: /date$/,
+      },
+    },
+  },
+} as Meta;
+
+export const Basic: Story<DateTimeProps> = (args) => <DateTime {...args} />;
+Basic.args = {
```

Comment:
Maybe an overview of the possible variants would be helpful, both for DateTime and RelativeDateTime? Or it's overkill?

Reply:
I think it's a good idea to always have some Overview! I will add it